pkgs/os-specific/linux/iptables/default.nix

```diff
@@ -1,32 +1,49 @@
-{ stdenv, fetchurl, bison, flex, pkgconfig, pruneLibtoolFiles
-, libnetfilter_conntrack, libnftnl, libmnl, libpcap }:
+{ stdenv, fetchurl, pkgconfig, pruneLibtoolFiles, flex, bison
+, libmnl, libnetfilter_conntrack, libnfnetlink, libnftnl, libpcap
+, modeCompat ? false
+}:
+
+with stdenv.lib;
 
 stdenv.mkDerivation rec {
-  pname = "iptables";
   version = "1.8.3";
+  pname = "iptables";
+  name = pname + "-" + (if modeCompat then "compat" else "legacy") + "-" + version;
```

Comment:
I don't see the value of changing the name.

Reply:
Fixed.